src/mui-theme-provider.jsx

```diff
@@ -0,0 +1,23 @@
+import {Component, PropTypes} from 'react';
+
+class ThemeProvider extends Component {
+  getChildContext() {
+    return {
+      muiTheme: this.props.muiTheme,
+    };
+  }
+  render() {
+    return this.props.children;
+  }
+}
+
+ThemeProvider.propTypes = {
+  children: PropTypes.element,
+  muiTheme: PropTypes.object,
+};
+
+ThemeProvider.childContextTypes = {
```

Comment:
Yeah we should be able to do that if we have the right babel stage/preset. I'll give it a try and if it works I'll move it!